packages/workflows-shared/src/binding.ts

```diff
@@ -34,22 +31,47 @@ export class WorkflowBinding extends WorkerEntrypoint<Env> implements Workflow {
 			}
 		);
 
-		return new WorkflowHandle(id, stub);
+		const handle = new WorkflowHandle(id, stub);
+		return {
+			id: id,
+			pause: handle.pause.bind(handle),
+			resume: handle.resume.bind(handle),
+			terminate: handle.terminate.bind(handle),
+			restart: handle.restart.bind(handle),
+			status: handle.status.bind(handle),
+		};
```

Comment:
Why this change? Is `WorkflowHandle` not an `RpcStub`?

Reply:
The usage of a `RcpStub` make it mandatory to `await` every parameter get,  (such as `instance.id`), which is not the same as in Workflows production deployment. This change allow us to not require the `await` on the gets, allowing customers to run the exact same code on wrangler local dev and Workflows production.
Notice that this `binding.ts` logic is an emulation of what we have in Workflows production environment.

Reply:
In production, we create a Object in workerd that returns a handle. We started out with an `RpcTarget` here but @bruxodasilva found a bug in that locally that implies every property access is async (which is not true in production). This emulates that without the other services in production.